index.html

```diff
@@ -10,10 +10,98 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+    
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    
+    <main>
+        <header class="header">
+            <img class="fav_ico" src="img/karma-logo.svg"/>
+            
+            <div>          
+                <nav class="nav_ul">
+                    <a href="#">Meet Karma</a>
+                    <a href="#">How it Works</a>
+                    <a href="#">Store</a>
```

Comment:
You linked the button in the new page section to the store page, just need to update this item to link to the store as well :+1: